extension-manifest-v3/src/pages/onboarding/index.js

```diff
@@ -37,5 +41,12 @@ define({
     ></ui-onboarding>`,
 });
 
-// Update options showAt timestamp
-store.set(Options, { onboarding: { shownAt: Date.now() } });
+store.ready(Options).then(() => {
```

Comment:
It is wrong, you are not using `store.ready()` with the definition, but the model instance. Use `store.resolve(Options)` if you need to read some options before updating.